server/pulp/server/webservices/views/util.py

```diff
@@ -61,7 +82,8 @@ def wrapper(*args, **kwargs):
             return func(*args, **kwargs)
 
         try:
-            request.body_as_json = json.loads(request.body)
+            request_json = json.loads(request.body)
+            request.body_as_json = _ensure_input_encoding(request_json)
```

Comment:
I think the important thing is to not include statements in try blocks accidentally. If you want multiple statements to each be guarded for the same exception type being raised then I think that makes sense.

A bit unrelated, but the thing we should never write is

```
try:
    <thing>
except Exception:
    pass
```

Catching all Exception types and silencing them is a great way to cause really hard to find bugs. That is one rule that I think we should be strict on.